docs/latest/essentials/translating.md

```diff
@@ -0,0 +1,254 @@
+}
+
+export default withTranslation('MyNameSpace')(MyComponent);
+```
+
+> Important: if you are using React outside the OHIF Viewer, check the
+> [I18nextProvider](#using-outside-of-ohif-viewer) section, `withTranslation`
+> HOC doesnt works without a I18nextProvider
+
+##### Using Hooks
+
+Also, it's possible to get the `t` tool using
+[React Hooks](https://react.i18next.com/latest/usetranslation-hook), but it
+requires at least React > 16.8 😉
+
+#### Using outside of OHIF viewer
+
+OHIF Viewer already sets a main
+[I18nextProvider](https://react.i18next.com/latest/i18nextprovider) connected to
+the shared i18n instance from `@ohif/i18n`, all extensions inside OHIF Viewer
+will share this same provider at the end, you don't need to set new providers at all.
+
+But, if you need to use it completely outside of OHIF viewer, you can set the
+I18nextProvider this way:
+
+```js
+import i18n, { I18nextProvider } from '@ohif/i18n';
+import App from './App';
+
+<I18nextProvider i18n={i18n}>
+  <App />
+</I18nextProvider>;
+```
+
+After setting `I18nextProvider` in your React App, all translations from
+`@ohif/i18n` should be available following the basic [With React](#with-react) usage.
+
+---
+
+#### Without React
+
+When needed, you can also use available translations _without React_.
+
+E.g.
+
+```js
+import { t } from '@ohif/i18n';
+console.log(t('my translated text'));
+console.log(t('$t(Common:Play) my translated text'));
+```
+
+---
+
+# Main Concepts While Translating
+
+## - Namespaces
+
+Namespaces are being used to organize translations in smaller portions, combined
+semantically or by use. Each `.json` file inside `@ohif/i18n` npm package
+becomes a new namespace automatically.
+
+- Buttons: All buttons translations
+- CineDialog: Translations for the toll tips inside the Cine Player Dialog
+- Common: all common jargons that can be reused like `t('$t(common:image)')`
+- Header: translations related to OHIF's Header Top Bar
+- MeasurementTable - Translations for the react-viewerbase Measurement Table
+- UserPreferencesModal - Translations for the react-viewerbase Preferences Modal
+
+### How to use another NameSpace inside the current NameSpace?
+i18next provides a parsing feature able to get translations strings from any NameSpace,
+ like this following example getting data from `Common` NameSpace:
+```
+$t(Common:Reset)
+```
+
+## - Extending Languages in @ohif/i18n
+
+Sometimes, even using the same language, some nouns or jargons can change according to 
+the country, states or even from Hospital to Hospital. 
+
+In this cases, you don't need to set an entire language again, you can extend languages creating a new folder inside a pre existent language folder and @ohif/i18n will do the hard work.
+
+This new folder must to be called with a double character name, like the `UK` in the following file tree:
+
+<img src="/assets/img/ohif-i18n-extending-files-tree.png" alt="Files Tree for Extending Purpouses" style="margin: 0 auto;" />
```

Comment:
This isn't showing up for me in the docs: https://deploy-preview-526--ohif.netlify.com/essentials/translating.html

Reply:
replaced for code made file tree.